mission/finite_state_machine/scripts/fsm_helper.py

```diff
@@ -5,33 +5,32 @@
 from smach import StateMachine, Sequence, Concurrence, cb_interface, CBState
 from smach_ros import SimpleActionState
 from geometry_msgs.msg import Point, Quaternion
-from vortex_msgs.msg import MoveGoal, MoveAction
 from tf.transformations import quaternion_from_euler
+from move_base_msgs.msg import MoveBaseAction, MoveBaseGoal
+from vortex_msgs.msg import LosPathFollowingAction, LosPathFollowingGoal
 
-move_action_server = '/guidance/move'
-# maybe create class for this? 
-# something to signal that the **_move functions are used for crosstalk
-# between the FSM and the Guidance systems.
-# rename the file accordingly too; put in own folder
 
-def dp_move(x, y, z=-0.5, yaw_rad=0):
-    goal = MoveGoal()
+guidance_interface_dp_action_server="/guidance_interface/dp_server"
+guidance_interface_los_action_server="/guidance_interface/los_server"
 
```

Comment:
I think these should be defined in the drone config, since these params will be easily lost in a helper function this deep in the file tree.